src/internal/svg/disconnectSvg.tsx

```diff
@@ -6,16 +6,16 @@ export const disconnectSvg = (
     aria-label="ock-disconnect-svg"
     width="100%"
     height="100%"
-    viewBox="0 0 20 20"
+    viewBox="0 0 16 20"
```

Comment:
Do we have a story for this component on Storybook?

Reply:
No, we do not have a storybook for any of our internal SVGs. Should I try to make one?

Reply:
I have a Fix Storybook task on Linear but it's getting deprioritized for tasks such as fix landing page and the upcoming Swap Component changes https://linear.app/coinbase/issue/BOE-313/fix-storybook

Reply:
My comment was more about. Go make a storybook page for every component you touch :) 

So in this case you touch this component, let's have a story for this.

Reply:
TIL you can add a story for an SVG! Added!